pyaedt/emit_core/results/revision.py

```diff
@@ -662,3 +663,36 @@ def protection_level_classification(
             power_matrix.append(rx_powers)
 
         return all_colors, power_matrix
+
+    def get_emi_category_filter_enabled(self, category: EmiCategoryFilter) -> bool:
+        """Get whether the EMI category filter is enabled.
+
+        Parameters
+        ----------
+        category : :class:`EmiCategoryFilter`
+            The EMI category filter.
+
+        Returns
+        -------
+        bool
+            ``True`` when the EMI category filter is enabled.
+        """
+        if self.emit_project._aedt_version < "2024.1":  # pragma: no cover
+            raise RuntimeError("This function only supported in AEDT version 2024.1 and later.")
```

Comment:
```suggestion
            raise RuntimeError("This function is only supported in AEDT version 2024 R1 and later.")
```
A complete sentence requires a verb. User-facing documentation should use a version format like 2024 R1 (with a space).